src/main/java/com/github/crimsondawn45/fabricshieldlib/lib/object/FabricShieldDecoratorRecipe.java

```diff
@@ -66,9 +64,8 @@ public ItemStack craft(CraftingInventory craftingInventory) {
             if (!itemStack3.isEmpty()) {
                 if (itemStack3.getItem() instanceof BannerItem) {
                     itemStack = itemStack3;
-                } else if (itemStack3.getItem() instanceof FabricShield) {
-                    FabricShield theShieldItem = ((FabricShield) itemStack3.getItem());
-                    if (theShieldItem.supportsBanner()){
+                } else if (itemStack3.getItem() instanceof FabricShield theShieldItem) {
+                    if (theShieldItem.supportsBanner()) {
```

Comment:
the redundant variable should stay as a precaution for crashes/errors

Reply:
Here I just used a pattern variable. You can see that the variable is inside the else if parenthesis.
[For more information about the pattern variable](https://www.agileconnection.com/article/using-new-pattern-matching-feature-java-16)